js/store/reducers.js

```diff
@@ -32,12 +40,55 @@ function itemsFetching(state) {
 }
 
 function itemsFetched(state, { value }) {
+  var ids = [];
+  const toCache = value.reduce((result, item) => {
+    result[item.id] = createItem(FETCHED, item);
+    ids.push(item.id);
+    return result;
+  }, {});
```

Comment:
I don't want to filter down the list, I want to transform it into an Array of Strings and an object. I think I could use a `.map()`, return just the ids and create the `toCache` object inside that loop.